src/deluge/gui/ui/menus.cpp

```diff
@@ -536,6 +536,7 @@ Submenu globalReverbMenu{
     STRING_FOR_REVERB,
     {
         &globalReverbSendAmountMenu,
+		&reverbModelMenu,
```

Comment:
Small nit, as @soymonitus mentioned, can you make the reverbModelMenu first in the menu list.

So:

```
&reverbModelMenu,
&globalReverbSendAmountMenu,
```

Thx

Reply:
Original's model position has been changed to keep amount first